package.json

```diff
@@ -31,6 +31,8 @@
     "test:unit": "cross-env TEST_ENV=unit jest"
   },
   "dependencies": {
+    "fast-fuzzy": "1.10.10",
+    "graphemesplit": "2.4.4",
```

Comment:
Why was this dependency added?

Reply:
it's a dependency of fast-fuzzy

Reply:
We don't have to add that manually. `npm` will manage it and resolve the dependency. You can safely remove it and you'll see that npm still resolves it in `package-lock.json`.

Reply:
I don't think I added it manually. I just npm i fast-fuzzy.

Reply:
package.json dependencies objects are only populated when you explicitly install them:

```bash
$ npm init -y > /dev/null
$ npm i fast-fuzzy
$ cat package.json
{
  "name": "foo",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "keywords": [],
  "author": "",
  "license": "ISC",
  "dependencies": {
    "fast-fuzzy": "^1.10.10"
  }
}
```